### PR TITLE
RD actions for Kudos

### PR DESCRIPTION
**Pull request by konrad04-1989**

### CODE REVIEW

File: spec/system/admin/admin_kudo_system_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  # rubocop:disable RSpec/ExampleLength
+  # rubocop:disable RSpec/MultipleExpectations
+
+  it 'enables me to show and destroy kudos' do
+    visit admin_kudos_path
+    fill_in 'Email', with: admin_user.email
+    fill_in 'Password', with: admin_user.password
+    click_button 'Log in'
+
+    expect(page).to have_text(kudo.title && kudo1.title)
```
Comment (oskarlakner) [inline spec/system/admin/admin_kudo_system_spec.rb]: this one is a bit weird. I'd write 2 expect's

File: config/routes.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
   namespace :admin do
     root to: 'pages#dashboard'
+    resources :kudos
```
Comment (oskarlakner) [inline config/routes.rb]: as this is RD action you could limit it's routes by `, only: [:index, :other_method]`

**Review by oskarlakner — APPROVED** [review]

File: spec/system/admin/admin_kudo_system_spec.rb (hunk 1 above)
Comment (konrad04-1989) [reply spec/system/admin/admin_kudo_system_spec.rb]: Corrected in next commit. Thanks!

File: config/routes.rb (hunk 2 above)
Comment (konrad04-1989) [reply config/routes.rb]: introduced in the next commit. Thanks! ;)

**Pull request merged**